pkgs/by-name/nu/nuv/package.nix

```diff
@@ -0,0 +1,68 @@
+{ lib
+, stdenv
+, pkgs
+, callPackage
+, fetchFromGitHub
+, fetchurl
+, buildGoModule
+, makeWrapper
+, breakpointHook
+, jq
+, curl
+, kubectl
+, eksctl
+, kind
+, k3sup
+, coreutils
+}:
+
+let
+  branch = "3.0.0";
+  version = "3.0.1-beta.2405292059";
+in
+buildGoModule rec {
+  inherit branch version;
+
+  pname = "nuv";
+
+  src = fetchFromGitHub {
+    owner = "nuvolaris";
+    repo = "nuv";
+    rev = version;
+    sha256 = "sha256-MdnBvlA4S2Mi/bcbE+O02x+wvlIrsK1Zc0dySz4FB/w=";
+  };
+
```

Comment:
... like this!

```suggestion

  vendorHash = "sha256-JkQbQ2NEaumXbAfsv0fNiQf/EwMs3SDLHvu7c/bU7fU=";
  subPackages = [ "." ];
  
```

Reply:
these two lines are after src. I leaved them in alphabetical order. Let me know if it's ok